app/experimenter/kinto/diagrams/diagrams.sequence

```diff
@@ -0,0 +1,257 @@
+participant Experiment Owner
```

Comment:
Would be useful to have some notes hereabouts on how to re-generate the PNG renderings if/when any of this needs to change